invenio_notifications/config.py

```diff
@@ -1,9 +1,29 @@
 # -*- coding: utf-8 -*-
 #
 # Copyright (C) 2023 CERN.
+# Copyright (C) 2023 Graz University of Technology.
 #
 # Invenio-Notifications is free software; you can redistribute it and/or
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
 """Invenio module for notifications support."""
+
+from flask_babelex import gettext as _
+
+from .backends.email.backend import EmailNotificationBackend
+
+NOTIFICATIONS_BACKENDS = {
+    EmailNotificationBackend.id: EmailNotificationBackend,
```

Comment:
does it make sense to place the id explicitly here?

Reply:
It allows for easier lookup.
When registering a NotificationOp, the caller/service will have to provide some sort of id to know which notification back-end to use. Do you have any concerns with using the id in this case?

Reply:
Oh sorry. In the config it does not really makes sense though, you are right.